pages/learn/develop/container-contracts.md

```diff
@@ -61,9 +61,9 @@ requires:
 
 | Contract type  | Description                                                      | Valid from |
 |----------------|------------------------------------------------------------------|------------|
-| sw.supervisor  | Device Supervisor version                                        | 10.16.17   |
-| sw.l4t         | [L4T][l4t] version                                               | 10.16.17   |
-| hw.device-type | The [device type][device-type] as given by `BALENA_MACHINE_NAME` | 11.1.0     |
+| sw.supervisor  | Device Supervisor version (specified as `version`)               | 10.16.17   |
```

Comment:
To improve the effectiveness of this change, might I recommend adding an example here to show how you like this to be specified